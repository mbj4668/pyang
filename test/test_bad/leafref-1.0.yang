module leafref-1.0 {
  yang-version 1;
  namespace urn:example:ll;
  prefix ll;

  container x {
    leaf foo {
      type string;
    }
    leaf bar {
      type leafref {
        path "../foo";
        require-instance false; // error
      }
    }
  }
}
