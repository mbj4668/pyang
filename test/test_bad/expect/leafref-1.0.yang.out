leafref-1.0.yang:13: error: BAD_RESTRICTION
